thirdparty/build-thirdparty.sh

```diff
@@ -1026,12 +1032,14 @@ build_serdes() {
                 --LDFLAGS="-L ${TP_INSTALL_DIR}/lib -L ${TP_INSTALL_DIR}/lib64" \
                 --enable-static \
                 --disable-shared
-
+    OLD_BUILD_SYSTEM=${BUILD_SYSTEM}
+    BUILD_SYSTEM='make'
```

Comment:
no need to save to `$OLD_BUILD_SYSTEM`, directly use `make` in following two lines.